openedx.yaml

```diff
@@ -0,0 +1,9 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: adap
+oeps: {}
+openedx-release: {parent-repo: edx/edx-analytics-dashboard}
+owner: edx/analytics
+tags: [analytics]
```

Comment:
I know we already have a ton of these, but a more uniform use of yaml syntax would probably be a good idea.